boards/arm/nrf52810_pca10040/doc/nrf52810_pca10040.rst

```diff
@@ -0,0 +1,24 @@
+.. _nrf52810_pca10040:
+
+nRF52810-PCA10040
+#################
+
+Overview
+********
+
+The nrf52810_pca10040 board is a modified version of the :ref:`nrf52_pca10040`
+that enforces the limitations imposed by the nRF52810 IC, which is a
```

Comment:
I am wondering if we shall mention here that this solution combines limations of nrf52832 and nrf52810.
User shall be aware that PANs fixed in 52810 are still valid with this approach, hence intialization time will be extended.
Furthermore nr52810 has 10bit transfer buffers for SPIM, TWIM, UARTE but you can use only 8 bits because it is HW limation in nr52832.

Reply:
I think this is a rather generic comment, and we should not care about the details. I think we just need to stress that 810 is a reduced version of 832.